pkg/itemblock/actions/pvc_action.go

```diff
@@ -39,6 +39,8 @@ import (
 type PVCAction struct {
 	log      logrus.FieldLogger
 	crClient crclient.Client
+	// map[namespace]->[map[podName]->[]pvcVolumes]
+	nsPods map[string]map[string][]string
```

Comment:
Since `NewPVCAction` is searching pods by the given PVC, why not we make the map like  `map[namespace]->[map[pvcName]->[]pods]`?
This will be faster, as I can see below problem with the current code:
 `for pod, vols := range pods`
Since it needs to iterates PVCs, so it has to treat the pod map as array, and finally make the time complex as O(N). Actually, it is even slower as enumerating from a map is slower than from an array.

`for _, volume := range vols`
Again, it further enumerates PVCs for each pod.

With `map[namespace]->[map[pvcName]->[]pods]`, we almost only need to make a O(logN) search for PVC (Iterating pods for each PVC could be ignored since it is not normal cases for multiple pods sharing the same PVC).